### ISSUE TITLE
How do I add it to my macOS project?

### ISSUE THREAD

**Issue by mickaelhero** [fathyb/JSCameraKit#2]
Hello I have a macOS application written in swift. I have several chat webview which allow me to send voice or video messages.

How do I use it with my wkwebview?

How to inject the js script into the sites to link the communication between the webview and the site?

There is no documentation

**Comment by M4THYOU** [comment]
@mickaelhero Have you gotten anywhere with this? I'm working on trying to implement this as well, I'll let you know if I figure anything out

**Comment by mickaelhero** [comment]
@M4THYOU No, I didn’t succeed.
But I have successfully enabled webRTC using the webkit Private API. For the moment only the voice record works correctly, I work to make  work the video too. But the application cannot be uploaded to the store.
So the best solution is to go through JS.
If you succeed let me know. Otherwise I can give you my solution using the private API.

I see on your profile that there is your email, if you want we can send emails to work together on the solution?

**Comment by M4THYOU** [comment]
@mickaelhero Yes, I would love to work with you on a solution. In the meantime, would you be able to share your solution with the private API? And yes, you can reach me by the email address in my profile.

**Comment by mickaelhero** [comment]
@M4THYOU Email sent :)
